corehq/motech/repeaters/tests/test_models.py

```diff
@@ -46,17 +50,34 @@ def test_get_all_repeater_types():
         assert_in(name, types)
 
 
-class RepeaterTestCase(TestCase):
+class RepeaterTestCase(TestCase, DomainSubscriptionMixin):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(DOMAIN)
+        cls.addClassCleanup(clear_plan_version_cache)
+        cls.addClassCleanup(cls.domain_obj.delete)
+
+        # DATA_FORWARDING is on PRO and above
+        cls.setup_subscription(DOMAIN, SoftwarePlanEdition.PRO)
+        cls.addClassCleanup(cls.teardown_subscriptions)
 
     def setUp(self):
         super().setUp()
         url = 'https://www.example.com/api/'
-        self.conn = ConnectionSettings.objects.create(domain=DOMAIN, name=url, url=url)
+        self.conn = ConnectionSettings.objects.create(
+            domain=DOMAIN,
+            name=url,
+            url=url,
+        )
+        self.addCleanup(self.conn.delete)
         self.repeater = FormRepeater(
             domain=DOMAIN,
             connection_settings=self.conn,
         )
         self.repeater.save()
+        self.addCleanup(self.repeater.delete)
```

Comment:
I think these cleanups are unnecessary (and only make the tests slower) since SQL changes are automatically rolled back after each test.

Reply:
> since SQL changes are automatically rolled back

Really? That's useful! Has this always been true?

Reply:
Yes. It's a [Django `TestCase`](https://docs.djangoproject.com/en/5.0/topics/testing/tools/#django.test.TestCase) feature.